### DISCUSSION TITLE
Sideset updater

### DISCUSSION THREAD

**Discussion by lyyc199586** [idaholab/moose#26694] [Q&A Meshing]
Do we have a sideset updater now? Useful to enforce boundary conditions on time-varying boundaries

**Comment by dschwen** [comment]
You mean this: https://github.com/idaholab/moose/pull/26456 ? It's not merged yet. I still need to finish up the PR. The functionality should be complete, but it needs tests docs and SQA. Use it at your own risk.

**Reply by lyyc199586** [reply]
got it
